### ISSUE TITLE
[WebProfilerBundle] Unable to generate a URL for the named route "_profiler_open_file"

### ISSUE THREAD

**Issue by kunicmarko20** [symfony/symfony#26754]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | yes
| RFC?             | no
| Symfony version  | 4.0.7

After updating to 4.0.7 I am getting this error:

```
$ bin/console cache:clear --env=test

In srcTestDebugProjectContainerUrlGenerator.php line 106:

  Unable to generate a URL for the named route "_profiler_open_file" as such route does not exist.
```

@javiereguiluz pointed me to https://github.com/symfony/symfony/pull/26626, so I guess it is connected to that. Didn't have time to investigate more so I just downgraded to 4.0.6 and it works as expected.

**Comment by hjanuschka** [comment]
same here. @kunicmarko20 also disabling it in testing now

**Comment by NicolasDievart** [comment]
Same here on my project with Symfony 3.4.7

**Comment by p5mtamazlicaru** [comment]
Same here from 4.0.6 -> 4.0.7

**Comment by garak** [comment]
I can confirm.
Workaround: creaete a `config/test` directory and copy `web_profiler.yaml` from `config/dev` to it

**Comment by shulard** [comment]
Hello !

I got the same error after updating to 4.0.7. It seems that removing the `test/web_profiler.yaml` file "solve" the issue...

**Comment by TheKassaK** [comment]
Same here... Don't know how resolve this

**Comment by garak** [comment]
@TheKassaK see my previous comment

**Comment by TheKassaK** [comment]
Yes, you have to change the file config/bundles.php et remove the 'test' => true for the profiler : 

    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true],

And remove the file config/packages/test/web_profiler.yaml

And it works.... for now !

**Comment by kunicmarko20** [comment]
@TheKassaK that is how I did it.

**Comment by garak** [comment]
Removing the profiler for testing is suitable only if you never use it in your tests (that's not my case)

**Comment by fabpot** [comment]
Can you test #26758? Does it fix it?

**Comment by p5mtamazlicaru** [comment]
I don't know how to test it on sf4. :|

**Comment by fdyckhoff** [comment]
I added the changes manually. Looked like it works. At least the exception was gone. (Cache clear command for --env=test worked again)

**Comment by fabpot** [comment]
Thank you @fdyckhoff for checking the fix.

**Comment by cve** [comment]
@kunicmarko20 how did you downgrade to v4.0.6?

**Comment by elchris** [comment]
I just upgraded to Symfony 3.4.7 from Symfony 3.4.6:

 ~/cloud/katas/kata_tdd_php_symfony (demo-run-4-with-acceptance-and-auth)*$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 11 updates, 0 removals
- Updating twig/twig (v2.4.6 => v2.4.8): Downloading (100%)         
- Updating paragonie/random_compat (v2.0.11 => v2.0.12): Downloading (100%)     
**- Updating symfony/symfony (v3.4.6 => v3.4.7): Downloading (100%)**         
- Updating doctrine/doctrine-cache-bundle (1.3.2 => 1.3.3): Downloading (100%)  
- Updating symfony/phpunit-bridge (v3.4.6 => v3.4.7): Downloading (100%)       
- Updating guzzlehttp/guzzle (6.3.0 => 6.3.2): Downloading (100%)         
- Updating phpunit/phpunit (7.0.2 => 7.0.3): Downloading (100%)         
- Updating codeception/phpunit-wrapper (7.0.2 => 7.0.6): Downloading (100%)     
- Updating codeception/codeception (2.4.0 => 2.4.1): Downloading (100%)         
- Updating doctrine/dbal (v2.6.3 => v2.7.0): Downloading (100%)         
- Updating composer/ca-bundle (1.1.0 => 1.1.1): Downloading (100%) 

When I run phpunit against my test suite, my DefaultControllerTest:testIndex() fails with this exception:

./vendor/bin/phpunit
PHPUnit 7.0.3 by Sebastian Bergmann and contributors.

E.......................................................          56 / 56 (100%)

Time: 4.08 seconds, Memory: 48.00MB

There was 1 error:

1) Tests\AppBundle\Controller\DefaultControllerTest::testIndex
Symfony\Component\Routing\Exception\RouteNotFoundException: Unable to generate a URL for the named route "_profiler_open_file" as such route does not exist.

/Users/cholland/cloud/katas/kata_tdd_php_symfony/var/cache/test/appTestDebugProjectContainerUrlGenerator.php:36
/Users/cholland/cloud/katas/kata_tdd_php_symfony/vendor/symfony/symfony/src/Symfony/Component/Routing/Router.php:240

**Comment by elchris** [comment]
I have the breaking version on this branch:

https://github.com/elchris/kata_tdd_php_symfony/tree/demo-run-4-with-acceptance-and-auth-symfony-3.4.7

composer install
./vendor/bin/phpunit

**Comment by curry684** [comment]
Not sure if we're just weird, but *every* project I upgrade to 4.0.7 here is now failing CI tests because of this. 

Considering big projects are doing stuff like https://github.com/pamil/Sylius/commit/65fe5f12aa482a8c2a4b0880fc0b4ce5e1090ac8 I'd recommend releasing a 4.0.8 shortly, and possibly even yanking 4.0.7.

**Comment by Nek-** [comment]
**Your version of `symfony/security-bundle` is wrong.**

In `4.x` you need to use `sensiolabs/security-checker` (the command is drop in `symfony/security-bundle` 4.x).

**Comment by kunicmarko20** [comment]
If I see correctly, `4.0.8` is tagged, so you can upgrade :smile:
